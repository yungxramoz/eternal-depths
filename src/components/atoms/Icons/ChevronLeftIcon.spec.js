import { render, screen } from '@testing-library/react'
import React from 'react'
import ChevronLeftIcon from './ChevronLeftIcon'

describe('ChevronLeftIcon', () => {
  it('renders correctly', () => {
    render(<ChevronLeftIcon />)
    expect(screen).toMatchSnapshot('ChevronLeftIcon')
  })
})
