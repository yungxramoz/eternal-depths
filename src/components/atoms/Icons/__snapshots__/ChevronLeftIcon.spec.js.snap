// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChevronLeftIcon renders correctly: ChevronLeftIcon 1`] = `
<body>
  <div>
    <svg
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M16 5v2h-2V5h2zm-4 4V7h2v2h-2zm-2 2V9h2v2h-2zm0 2H8v-2h2v2zm2 2v-2h-2v2h2zm0 0h2v2h-2v-2zm4 4v-2h-2v2h2z"
        fill="currentColor"
      />
    </svg>
  </div>
</body>
`;
